pages/phases/job-analysis/defining-proficiency-levels.md

```diff
@@ -6,48 +6,91 @@ title: Defining Proficiency Levels and Required Experience
+experience doing so. Anticipates the need for metrics and analysis early in the product
+and project discovery and design process, and carries metrics through iterations.
+
+### Four levels
+
+Use four levels when you may want to leverage this work for other grades in the future; the same set of competencies and proficiencies can be used for different grade levels because roles that require more seniority set the required proficiencies higher than junior roles. It may also help to define four levels before deciding which level is appropriate for the grade in question. If you use four levels, we recommend these titles for each level:
+
+1. Novice
+2. Competant
+3. Advanced
+4. Expert
+
+Once proficiency levels are defined, HR and the SMEs will decide at which level you would require applicants to have one year of experience in order to qualify for the grade in question. For example, they made decide that to qualify for a GS13, applicants must have one year experience at the “Advanced” level. See this example:
+
+#### Example (for an analytical competancy)
+
+* **Novice:** Unable to break down problems or only talks through problems at a high
+level. Does not make data-driven decisions, or actively eschews data in decision-making.
+* **Competant:** Displays some critical thinking and problem-solving abilities. May start to break
+a problem down in component parts, but not completely. Able to identify basic product
+or project metrics but does not connect these to business or user value.
+* **Advanced (Required):** Breaks problems down into component parts. Displays repeated
+experience in qualitative and quantitative analysis. Defines product or project metrics
+beyond the basics and ties these metrics to decisions.
+* **Expert:** Brings analytical thinking to everything they do, and has a range and depth of
+experience doing so. Anticipates the need for metrics and analysis early in the product
+and project discovery and design process, and carries metrics through iterations.
+
+### Generic proficiency levels
+
+**Novice:** Applies the competency in simple situations, requiring close and extensive guidance. Has the level of experience gained in a classroom and/or experimental scenarios or as a trainee on-the-job. Aware of terminology, concepts, principles, and issues related to this competency.
+
+> Novices have had little experience of the situation in which they are expected to perform. Their initial learning about the situation is in terms of objective attributes--those that are measurable. These are features of the task world that can be recognized without situational experiences. Novices are limited in their understanding to context-free rules that guide action--this means their behavior is limited and inflexible.
+
+**Basic:** Applies the competency in somewhat difficult situations, requiring frequent guidance. Familiar with and can discuss terminology, concepts, principles, and issues related to this competency.
```

Comment:
@wslack -- "Basic" is not a level identified above. Why are we adding a new level between "Novice" and "Competent?" I also really don't love the term "Basic" here, as it's very generic.

Reply:
The NIH model and the other I linked both have 5 levels. I don't like "basic" either and we can totally pull it if you want. Alternatively, we could cut "novice" since I don't think anyone will ever use it to qualify someone and start with this level, though I might want to rename it. (NIH said "basic" - the other said "advanced beginner"

Reply:
Agreed. Pulling Basic and matching the 4-level model above. Showing a 2-level, 4-level, and 5-level model on the same page is just too much complexity.